File: Funciones/10. Mayor_igual_numeros/js/mayor.js
/**
 * Mayor igual de dos números
 * Autor: Santiago GM
 * 03/04/2024
 */

let numUno;
let numDos;
let num;
//Como paramétro

function mayorIgual(pnum) {
  num = pnum;
  return num;
}
function validar(pnumUno, pnumDos) {
  numUno = pnumUno;
  numDos = pnumDos;

  if (numUno == numDos) {
    return "Los número son iguales";
  } else if (numUno > numDos) {
    return `El número uno es mayor que el dos. El uno es ${numUno} y el dos es ${numDos}`;
  } else {
    return `El número dos es mayor que el uno. El uno es ${numDos} y el dos es ${numUno}`;
  }
}
//Como Expresión
const mayorIgualExp = function (pnum) {
  num = pnum;
  return num;
}
const validarExp = function (pnumUno, pnumDos) {
  if (numUno == numDos) {
    return "Los número son iguales";
  } else if (numUno > numDos) {
    return `El número uno es mayor que el dos. El uno es ${numUno} y el dos es ${numDos}`;
  } else {
    return `El número dos es mayor que el uno. El uno es ${numDos} y el dos es ${numUno}`;
  }
}

File: python/8. mayor_dos_numeros/utils/mayor.py
# Mayor de dos números
# Autor: Santiago GM


#como paramétro
def validar(num_uno_p, num_dos_p) :
  num_uno, num_dos = num_uno_p, num_dos_p

  if (num_uno > num_dos) :
    return f"El número uno es mayor{num_uno} "
  else:
    return f"El número dos es mayor {num_dos}"

# como Expresión
validar_lamb = lambda num_uno, num_dos: f"El número uno es mayor {num_uno}" if num_uno > num_dos else f"El número dos es mayor {num_dos}"